<?xml version="1.0"?>
<launch>
  <node pkg="serial_gps" type="gps_node1" name="gps_node1" output="screen">
        <param name="enable_gps" value="true"/> 
        <param name="gps_port" value="/dev/ttyUSB0"/>
        <param name="gps_baud" value="115200"/>
       <param name="ref_lat" value="10.904197   "/>
       <param name="ref_lon" value="106.797314  "/>
  </node>
</launch>

<!-- 10.904197   106.797314 đầu -->
<!-- 10.903798  106.798681 đuôi -->